server/validations/userValidations.js

```diff
@@ -19,7 +19,7 @@ class Validations {
     } catch (error) {
       res.status(400).send({

```

Comment:
please adopt the response handler to handle all responses.. try as much not to return `res.status().send()` . all the time